docs/faq.md

```diff
@@ -99,29 +150,34 @@ evolve without breaking existing clients: simply adding a field to a response wi
 Connect clients and servers will ignore unknown fields, provided that the
 underlying implementation allows us to do so.
 
+### How do I customize serialization errors returned by the Connect server?
+
+You can customize the error message by providing a different `Codec`. Code
+and details can't be customized.
+
 ## Go
 
 ### Why use generics?
 
 Generic code is inherently more complex than non-generic code. Still, introducing
-generics to `connect-go` eliminated two significant sources of complexity:
+generics to Connect-Go eliminated two significant sources of complexity:
 
 - Generics let us generate less code, especially for streaming RPCs &mdash; if
   you're willing to write out some long URLs, it's now just as easy to use
-  Connect without `protoc-gen-connect-go`. The generic stream types, like
+  Connect without `protoc-gen-Connect-Go`. The generic stream types, like
```

Comment:
Did we decide to use Connect-Go instead of `connect-go`? The latter is idiomatic naming for Go projects. Anyhow, this needs to be `protoc-gen-connect-go` because it's a plugin name, a command where casing matters.